.pre-commit-config.yaml

```diff
@@ -1,4 +1,11 @@
 repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.8.2
+    hooks:
+      - id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
```

Comment:
Is exit-non-zero-on-fix needed for pre-commit? I thought pre-commit failed if anything changed on disk?

Reply:
Ah, right you are! Removed :)